src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -692,6 +693,10 @@ function cancelSelection() {
 		@include meta.load-css('@vuepic/vue-datepicker/dist/main.css');
 	}
 
+	.vue-date-time-picker--clearable :deep(.dp__input) {
+		padding-inline-end: var(--default-clickable-area);
```

Comment:
Should be not just update the `--dp-input-padding` variable here?

Reply:
My reasoning was just that the variables are public API but the class names could break someday.